package.json

```diff
@@ -34,6 +36,7 @@
     "app-root-path": "^2.0.1",
     "brolog": "^1.2.6",
     "encoding-down": "^2.3.1",
+    "jsdoc-to-markdown": "^3.0.0",
```

Comment:
This is not right.

you need to add `jsdoc-to-markdown` to `devDependencies` instead of `dependencies` by run `npm install jsdoc-to-markdown --save-dev`